File: Melo/Melo.Models/Models/Playlist/PlaylistResponse.cs
namespace Melo.Models
{
	public class PlaylistResponse
	{
		public int Id { get; set; }

		public string? Name { get; set; }

		public string? Playtime { get; set; }

		public int? SongCount { get; set; }

		public List<PlaylistSongResponse> Songs { get; set; } = new List<PlaylistSongResponse>();
	}
}

File: Melo/Melo.UI/melo_mobile/lib/models/playlist_response.dart
class PlaylistResponse {
  final int id;
  final String? name;
  final int? songCount;
  final String? playtime;

  PlaylistResponse({
    required this.id,
    this.name,
    this.songCount,
    this.playtime,
  });

  factory PlaylistResponse.fromJson(Map<String, dynamic> json) {
    return PlaylistResponse(
      id: json['id'],
      name: json['name'],
      songCount: json['songCount'],
      playtime: json['playtime'],
    );
  }
}
